core/terraform/locals.tf

```diff
@@ -12,4 +12,24 @@ locals {
 
   # https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-configure-firewall#allow-requests-from-the-azure-portal
   azure_portal_cosmos_ips = "104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26"
+
+  # we define some zones in core despite not used by the core infra because
+  # it's the easier way to make them available to other services in the system.
+  private_dns_zone_names_non_core = toset([
+    "privatelink.purview.azure.com",
+    "privatelink.purviewstudio.azure.com",
+    "privatelink.sql.azuresynapse.net",
+    "privatelink.dev.azuresynapse.net",
+    "privatelink.azuresynapse.net",
+    "privatelink.dfs.core.windows.net",
+    "privatelink.azurehealthcareapis.com",
+    "privatelink.dicom.azurehealthcareapis.com",
+
```

Comment:
is there a reason for this empty line?

Reply:
there was but I don't remember. I'll remove it.